src/pages/Movie/TopInfo/AddToListPopover/component.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Popover, Button, Icon } from 'antd';
+
+const AddToListComponent = ({
+  list,
+  addToList,
+  popoverVisible,
+  handleVisible
+}) => (
+  <>
+    <Popover
+      title="Add movie to list"
+      trigger="click"
+      visible={popoverVisible}
+      onVisibleChange={handleVisible}
+      content={(
+        <>
+          <div>
+            <Button type="link">Create new list ...</Button>
+          </div>
+          {list && list.map((item) => (
+            <div key={item.id}>
+              <Button
+                type="link"
+                onClick={() => addToList(item.id)}
```

Comment:
Don't use inline functions inside components, you can refactor this with creation separate container for button and handle onClick in the button's container

Reply:
Then we will have more than 3 levels and it is against the [component guide](https://docs.google.com/document/d/1Cvs9QLqpg9TWHygMETVU8ukdd2JXVogwTPeL0cA6ypA/edit).

Reply:
Done. I have added one more nesting as we decided.